intern-book-app/src/app/consts/bookList.ts

```diff
@@ -17,3 +17,5 @@ export const bookList: Book[] = [
   }
 ];
 
+
+
```

Comment:
There are useless lines.

You can resolve with settings of VSCode
- files.insertFinalNewline
- trimFinalNewlines
- files.trimTrailingWhitespace

Check here.
https://k-koh.hatenablog.com/entry/2020/01/28/194905